dartagnan/src/main/java/com/dat3m/dartagnan/wmm/Wmm.java

```diff
@@ -39,7 +39,7 @@ public static class Config {
 
     // These relations are part of every memory model (even the "empty" one) because they are
     // necessary to specify the anarchic program semantics.
-    public final static ImmutableSet<String> ANARCHIC_CORE_RELATIONS = ImmutableSet.of(CO, RF, RMW);
+    public final static ImmutableSet<String> ANARCHIC_CORE_RELATIONS = ImmutableSet.of(CO, RF, AMO, LXSX);
```

Comment:
I think the `AMO` can go because it is a purely static relation and carries no intrinsic semantics (unlike `LXSX` which can cause store failure on mismatching addresses).